### PR TITLE
Add Cobot interview article

### PR DESCRIPTION
**Pull request by ma06484**

### CODE REVIEW

File: _news/2021-07-18-cobot-woco-interview.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+tags: works-councils
+---
+
+The following is the first of a planned series of interviews with Works Councils (*Betriebsräte*) in Berlin tech companies, conducted by Berlin TWC's content team.
+
+For the first episode, we are very happy to hear from the recently elected Works Councillor at Cobot, a small Berlin software company.
+
+**What company do you work for and since when does a Works Council exist/are you part of?**
+
+I work for Cobot, which elected its first Works Councillor (me!) in October 2020.
```
Comment (shushugah) [inline _news/2021-07-18-cobot-woco-interview.md:12]: I find the `"Me"` identifier a bit evasive. I can understand the desire not to state name, given how small the company is, and would suggest adding an explanation for this.

File: _news/2021-07-18-cobot-woco-interview.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Works Council interview series #1: Cobot"
+tags: works-councils
+---
+
+The following is the first of a planned series of interviews with Works Councils (*Betriebsräte*) in Berlin tech companies, conducted by Berlin TWC's content team.
+
+For the first episode, we are very happy to hear from the recently elected Works Councillor at Cobot, a small Berlin software company.
```
Comment (shushugah) [inline _news/2021-07-18-cobot-woco-interview.md]: Add a hyperlink to Cobot? https://www.cobot.me/en

File: _news/2021-07-18-cobot-woco-interview.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Works Council interview series #1: Cobot"
+tags: works-councils
+---
+
+The following is the first of a planned series of interviews with Works Councils (*Betriebsräte*) in Berlin tech companies, conducted by Berlin TWC's content team.
```
Comment (shushugah) [inline _news/2021-07-18-cobot-woco-interview.md]: Change TWC to Tech Workers Coalition for SEO purposes :P

**Comment by shushugah** [comment]
Would it make sense to tag this article with `works-councils` so it shows up underneath as well? https://techworkersberlin.com/news/2020-03-13-beyond-quitting-babbel-employees-elect-works-council/

File: _news/2021-07-18-cobot-woco-interview.md (hunk 1 above)
Comment (ma06484) [reply _news/2021-07-18-cobot-woco-interview.md:12]: Would you suggest to remove the `me!`, and/or just put something in the intro? Do you have a concrete formulation in mind, like e.g. "who would like to stay anonymous"?

**Pull request merged**